nifi-nar-bundles/nifi-framework-bundle/nifi-framework/nifi-web/nifi-web-ui/src/main/webapp/js/nf/users/nf-users-table.js

```diff
@@ -524,6 +732,10 @@ nf.UsersTable = (function () {
                 markup += '<div title="Remove" class="pointer delete-user fa fa-trash"></div>';
             }
 
+            if (nf.Common.isDefinedAndNotNull(dataContext.component.accessPolicies)) {
```

Comment:
Should probably show the key when the accessPolicies are not empty. I think its just `!nf.Common.isEmpty(dataContext.component.accessPolicies)`.